### PR TITLE
update location issue tracker

### PR DESCRIPTION
**Pull request by Fuseteam**

### CODE REVIEW

**Comment by doniks** [comment]
Mhm. Are you quite sure? When I go to https://github.com/ubports/ubuntu-touch I see no indication of a redirection/deprecation/etc ...

**Comment by jonnius** [comment]
> Mhm. Are you quite sure? When I go to https://github.com/ubports/ubuntu-touch I see no indication of a redirection/deprecation/etc ...

I see a message:

>  This repository has been archived by the owner on Dec 12, 2023. It is now read-only.

File: contribute/bugreporting.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ This might seem obvious, but it's easy to miss. Go to (Settings - Updates) and m
 Check if the bug is already reported
 ------------------------------------
 
-Open up the bug tracker for `ubuntu-touch <https://github.com/ubports/ubuntu-touch>`_.
+Open up the bug tracker for `ubuntu-touch <https://gitlab.com/ubports/ubuntu-touch>`_.
```
Comment (jonnius) [inline contribute/bugreporting.rst:14]: Let's use the link to the issue tracker: https://gitlab.com/ubports/ubuntu-touch/-/issues

**Comment by doniks** [comment]
> > Mhm. Are you quite sure? When I go to https://github.com/ubports/ubuntu-touch I see no indication of a redirection/deprecation/etc ...
> 
> I see a message:
> 
> > This repository has been archived by the owner on Dec 12, 2023. It is now read-only.

Ah yeah, I see it now

File: contribute/bugreporting.rst (hunk 1 above)
Comment (doniks) [reply contribute/bugreporting.rst:14]: The advantage of the other link is it shows the Readme and instructions there

Comment (jonnius) [reply contribute/bugreporting.rst:14]: True. Then let's keep it as proposed in this PR.

**Review by doniks — APPROVED** [review]

**Pull request merged**